File: bot-base-client/src/main/java/gg/mineral/bot/base/lwjgl/util/glu/Project.java
package gg.mineral.bot.base.lwjgl.util.glu;

import gg.mineral.bot.impl.config.BotGlobalConfig;

public class Project {

    public static void gluPerspective(float fovy,
            float aspect,
            float zNear,
            float zFar) {
        if (BotGlobalConfig.isHeadless())
            return;

        org.lwjgl.util.glu.GLU.gluPerspective(fovy, aspect, zNear, zFar);
    }

}

File: bot-lwjgl-bridge/src/main/kotlin/gg/mineral/bot/lwjgl/util/glu/Project.kt
package gg.mineral.bot.lwjgl.util.glu

import gg.mineral.bot.impl.config.BotGlobalConfig
import org.lwjgl.util.glu.Project

object Project {
    @JvmStatic
    fun gluPerspective(
        fovy: Float,
        aspect: Float,
        zNear: Float,
        zFar: Float
    ) {
        if (BotGlobalConfig.headless) return

        Project.gluPerspective(fovy, aspect, zNear, zFar)
    }
}
